contracts/auth/AxelarAuthWeighted.sol

```diff
@@ -13,7 +13,7 @@ contract AxelarAuthWeighted is Ownable, IAxelarAuthWeighted {
 
     uint256 internal constant OLD_KEY_RETENTION = 16;
 
-    constructor(bytes[] memory recentOperators) {
+    constructor(bytes[] memory recentOperators, address owner) Ownable(owner) {
```

Comment:
```suggestion
    constructor(bytes[] memory recentOperators) Ownable(msg.sender) {
```
avoids breaking the deployment this way, and it can be transferred later as usual